jedi_vim.py

```diff
@@ -417,9 +416,6 @@ def rename():
                 vim.current.window.cursor = r.start_pos
                 vim_command('normal! cw%s' % replace)
 
-            result = new_buffer(window_path)
-            if not result:
-                return
             vim.current.window.cursor = cursor
```

Comment:
@davidhalter  removing those lines fixes the problem when using the minimal vimrc in this [comment](https://github.com/davidhalter/jedi-vim/issues/362#issue-57663089). I don't know if it breaks something else (I've using jedi-vim with this change without any problem for the past few weeks).